include/cling/Interpreter/InvocationOptions.h

```diff
@@ -14,16 +14,30 @@
 #include <vector>
 
 namespace cling {
+
+  class CompilerOpts {
```

Comment:
Could you add a documentation here? How this is different from InvocationOptions, why did you decide to not extend it?